static/js/components/ChannelBreadcrumbs.js

```diff
@@ -12,10 +12,10 @@ const ChannelBreadcrumbs = (props: BreadCrumbProps) => {
   const { channel } = props
 
   return (
-    <div>
+    <div className="breadcrumbs">
       <Link to="/">Discussions</Link>&nbsp;
       <span>&gt;</span>&nbsp;
-      <Link to={`/channel/${channel.name}`}>{channel.title}</Link>
+      {channel.title}
```

Comment:
This should probably remain a `<Link />` - we show the `ChannelBreadcrumbs` component on the page for each post, so it supplies a link back to the subreddit the post is from.